spec/spec_helper.rb

```diff
@@ -0,0 +1,7 @@
+require 'simplecov'
+SimpleCov.start
+
+require_relative './game'
+require_relative './team'
+require_relative './stats'
+require_relative './league'
```

Comment:
require relative usually is called if your file is in the same directory.  Also how are you highlighting require relative in your previous statement?

Reply:
Hey, because of the futbol_spec_harness it says to use require_relative in our lib.  I'm still a little fuzzy on it. I couldn't fully remember what we were gonna name our classes, so I just made up some in that file.  Of course we can change the spec_helper later.

Reply:
@JCBeldo  I agree with changing this spec helper when we get to adding all the files! @amspears007 I think we can use `require_relative` as long as we give it the correct file path so for this case, it would be `'./lib/game'` because in relative to the `spec_helper`, `.` moves one directory up and `lib/game` leads us to the file! Again, something we can refactor once we make these lib files!